### PR TITLE
added new section with content

### PR DESCRIPTION
**Pull request by Dkingofcode**
**This pull request makes the following changes:**
* Fixes issue #7 

**Checklist:**
- [x] File or folder now contains changes as specified in the issue i worked on
- [x] I have linked the issue I worked on to this pull request submitted by me
- [x] I certify that I ran my checklist.

Ping @Ifycode/git-github-training-documentation

### CODE REVIEW

File: doc/contributors-guide/making-and-submitting-changes.md [hunk 1]
```diff
@@ -8,6 +8,25 @@ Add video here
 
 Add text content
 
+
+### Training repo: Add your changes using the GitHub UI
+
+> *Fork the training repo (Additional information: The default branch of the training repo is the main branch. So on the "Create a new fork" page, make sure to uncheck the "Copy the main branch only" checkbox before clicking on the "create fork" button. This will allow you to also have the develop branch, since you will need the develop branch for the next steps).
+
+
+> *Go to the develop branch of your fork.
```
Comment (Ifycode) [inline doc/contributors-guide/making-and-submitting-changes.md:17]: This is not formatted as a list

**Review by Ifycode — CHANGES_REQUESTED** [review]

File: doc/contributors-guide/making-and-submitting-changes.md (hunk 1 above)
Comment (Dkingofcode) [reply doc/contributors-guide/making-and-submitting-changes.md]: Thank you for the update ma. @Ifycode  I have changed it to a list.

**Comment by Dkingofcode** [comment]
Hi ma @Ifycode . I have removed the extra spaces as you instructed.

File: doc/contributors-guide/making-and-submitting-changes.md [hunk 2]
```diff
@@ -8,6 +8,17 @@ Add video here
...
+
+### Training repo: Add your changes using the GitHub UI
+
+- Fork the training repo (Additional information: The default branch of the training repo is the main branch. So on the "Create a new fork" page, make sure to uncheck the "Copy the main branch only" checkbox before clicking on the "create fork" button. This will allow you to also have the develop branch, since you will need the develop branch for the next steps).
+- Go to the develop branch of your fork.
+- Create a new branch from the develop branch and make your changes in the new branch you created.
+- Send a pull request pointing to the develop branch of the original training repo.
+- Wait for review of your changes from the repo admin(s).
+
+
```
Comment (Ifycode) [inline doc/contributors-guide/making-and-submitting-changes.md:20]: Just one space here biko

**Review by Ifycode — CHANGES_REQUESTED** [review]

**Comment by Dkingofcode** [comment]
Goodday ma, @Ifycode. The space under the training section has been removed.

File: doc/contributors-guide/making-and-submitting-changes.md [hunk 3]
```diff
@@ -8,6 +8,14 @@ Add video here
 
 Add text content
 
+### Training repo: Add your changes using the GitHub UI
+
+- Fork the training repo (Additional information: The default branch of the training repo is the main branch. So on the "Create a new fork" page, make sure to uncheck the "Copy the main branch only" checkbox before clicking on the "create fork" button. This will allow you to also have the develop branch, since you will need the develop branch for the next steps).
+- Go to the develop branch of your fork.
+- Create a new branch from the develop branch and make your changes in the new branch you created.
+- Send a pull request pointing to the develop branch of the original training repo.
+- Wait for review of your changes from the repo admin(s).
```
Comment (Ifycode) [inline doc/contributors-guide/making-and-submitting-changes.md:17]: Some things here in the list were links and you did not format them as links that the reader can click on. Check the formattings... Links and any other type of formatting, and add them.

**Review by Ifycode — CHANGES_REQUESTED** [review]

**Comment by Dkingofcode** [comment]
Thanks for the headsup ma, @Ifycode. I have added the important links

**Review by Ifycode — APPROVED** [review]

**Pull request merged**